pysat/utils/_core.py

```diff
@@ -410,6 +410,8 @@ def load_netcdf4(fnames=None, strict_meta=False, file_format=None,
         else:
             out = xr.open_mfdataset(fnames, combine='by_coords',
                                     decode_timedelta=decode_timedelta)
+        # Close any open links to file through xarray.
+        out.close()
```

Comment:
It seems a little weird that we can close the file so early and everything still works. So xarray must actually load everything from disk with the commands above. I wonder what the remaining open link to the file is for.

I would expect the close to be on line 435.

Reply:
The open commands should load everything from the file.  Moved as suggested in case it behaves differently on different systems.